components/organisms/Breadcrumb.js

```diff
@@ -0,0 +1,86 @@
+import { useContext, useEffect, useState } from "react";
+import { useRouter } from "next/router";
+import { faAngleRight } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import Link from "next/link";
+import { LanguageContext } from "../../context/languageProvider";
+import en from "../../locales/en";
+import fr from "../../locales/fr";
+
+/**
+ * breadcrumb component
+ */
+
+export default function BreadCrumbs(props) {
+  const router = useRouter();
+  const [breadcrumbs, setBreadcrumbs] = useState(null);
+
+  useEffect(() => {
+    if (router) {
+      const linkPath = router.asPath.split("/");
+      linkPath.shift();
+      const pathArray = linkPath.map((path, i) => {
+        return {
+          breadcrumb: path,
+          href: "/" + linkPath.slice(0, i + 1).join("/"),
+        };
+      });
+      setBreadcrumbs(pathArray);
+    }
+  }, [router]);
+  if (!breadcrumbs) {
+    return null;
+  }
+
+  const { items } = useContext(LanguageContext);
+  const language = items.language;
+  const t = language === "en" ? en : fr;
+  const countcrumbs = breadcrumbs.length;
+
+  return (
+    <nav aria-label="breadcrumbs">
+      <div>
+        <div className="inline">
+          <BreadLink
+            url="https://www.canada.ca/en/contact/index.html"
```

Comment:
This url brings us to the contact us page rather than Canada.ca's homepage. We have a link defined in our locales already for when we click on the header logo so let's use that instead.

Change this to ```url= {t.gocLink}``` (Currently, the link in fr.js is the same as en.js, we should change the french link to [https://www.canada.ca/en.html](https://www.canada.ca/en.html)

Reply:
Cool!  I'll pop that in tomorrow morning

Reply:
done!  did both....